src/connectors/paymentService.ts

```diff
@@ -556,18 +556,16 @@ export class PaymentService extends BaseService {
           targetType: undefined,
           targetId: undefined,
         })
-        .returning('id')
+        .returning('*')
```

Comment:
this line might not be necessary? I see you'll need only `trans.id`

Reply:
Updated to `.return('*')` because it has better compatibility, and is more consistent with the code in other places. For other code using `.return('*')`, the function is not breaking after being upgraded to knex v2.1.